Samples/Hast.Samples.Consumer/Program.cs

```diff
@@ -185,7 +134,7 @@ string GetArgument(string name)
                     typeof(Posit).Assembly,
                     typeof(ImmutableArray).Assembly,
                     // Only necessary for the F# sample.
-                    typeof(Microsoft.FSharp.Core.LiteralAttribute).Assembly
+                    typeof(LiteralAttribute).Assembly,

```

Comment:
Szándékosan nem akartam azt az elég nagy namespace-t behúzni egy C# fájlban.

Reply:
Miért, mi történik? Akkor 
`using Microsoft.FSharp.Core;` helyett lehet 
`using LiteralAttribute = Microsoft.FSharp.Core.LiteralAttribute;`?

Reply:
Elérhető lesz egyből gépelés közben minden az F#-ból, ami itt biztos nem kell. Ugye csak ez az egyetlen type, amit importálni akarunk onnan.

Lehetne az is, de nem látom különösebb értelmét.

Reply:
Ez továbbra is így van.

Reply:
Mármint micsoda? [Itt](https://github.com/Lombiq/Hastlayer-SDK/pull/59/commits/7d481f792fc586ef6cac26d2fc78c0c5bb72d69a#diff-85c928193ad3297f1d33851fb8f6c36362f03ded88cb5521539ab74b642c39baR16) lecseréltem a `using LiteralAttribute = Microsoft.FSharp.Core.LiteralAttribute;`-ra amit megbeszéltünk.

Reply:
Arra írtam, hogy "Lehetne az is, de nem látom különösebb értelmét."

Reply:
Az az értelme, hogy a code cleanup el akarja tüntetni a fölösleges namespace használatot.

Reply:
Milyen cleanup? Mert nem kellene, hiszen úgy helyes, és a VS nem is csinálja.

Reply:
Rider/Resharper féle code cleanup. Az esetek 99%-ban sokkal jobb mint a sima VS Organize Usings mert lepucolja a fölösleges inline namespace használatokat is amiket ezen az egy helyen kívül eddig soha nem szerettünk.